sql/scan.go

```diff
@@ -564,11 +600,6 @@ func (n *scanNode) prettyKey() string {
 			fmt.Fprintf(&buf, "/%v", t)
 		}
 	}
-	if n.colID > 0 {
-		// TODO(pmattis): This is inefficient, but does it matter?
-		col, _ := n.desc.FindColumnByID(n.colID)
-		fmt.Fprintf(&buf, "/%s", col.Name)
-	}
 	return buf.String()
```

Comment:
This is for debugging code (i.e. `EXPLAIN (DEBUG)`). I think convenience is more appropriate.